Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.py

```diff
@@ -1999,6 +2048,103 @@ def fetch_incidents(client: Client, last_run: Dict[str, Any], params: Dict[str,
+    access_key = client.access_key_creation(args)
+    transform_access_keys = {
+        'Id': access_key.get('id'),
+        'Secret Key': access_key.get('secretKey'),
+    }
+
+    markdown_table = tableToMarkdown('PrismaCloud Access Key Creation', transform_access_keys, headers=['Id', 'Secret Key'])
+    return CommandResults(
+        outputs_prefix="PrismaCloud.AccessKeys",
+        outputs_key_field='id',
+        outputs=access_key,
+        raw_response=access_key,
+        readable_output=markdown_table,
+    )
+
+
+def get_access_keys_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    return get_access_key_by_id(client, args) if args.get('access-key') else get_access_keys_list(client, args)
+
+
+def get_access_key_by_id(client: Client, args: Dict[str, Any]) -> CommandResults:
+    access_key = client.get_access_key_by_id(args)
+    transform_access_key = {
+        'ID': access_key.get('id'),
+        'Name': access_key.get('name'),
+        'Expires On': timestamp_to_datestring(access_key.get('expiresOn'), DATE_FORMAT)
+    }
+
+    markdown_table = tableToMarkdown('PrismaCloud Access Key', transform_access_key, headers=['ID', 'Name', 'Expires On'])
+    return CommandResults(
+        outputs_prefix="PrismaCloud.AccessKeys",
+        outputs_key_field='id',
+        outputs=access_key,
+        raw_response=access_key,
+        readable_output=markdown_table,
+    )
+
+
+def get_access_keys_list(client: Client, args: Dict[str, Any]) -> CommandResults:
+    access_keys = client.get_access_keys_list()
+    limit = int(args.get('limit', ACCESS_KEYS_LIST_DEFAULT_LIMIT))
+    limited_access_keys_list = access_keys[:limit]
+    transform_access_keys_list = [
+        {
+            'ID': access_key.get('id'),
+            'Name': access_key.get('name'),
+            'Created By': access_key.get('createdBy'),
+            'Created Timestamp': timestamp_to_datestring(access_key.get('createdTs'), DATE_FORMAT),
+            'Last Used Time': timestamp_to_datestring(access_key.get('lastUsedTime'), DATE_FORMAT),
+            'Status': access_key.get('status'),
+            'Expires On': timestamp_to_datestring(access_key.get('expiresOn'), DATE_FORMAT) if access_key.get(
+                'expiresOn') != 0 else '',
+            'Role id': access_key.get('role', {}).get('id'),
+            'Role Name': access_key.get('role', {}).get('name'),
+            'Role Type': access_key.get('roleType'),
+            'Username': access_key.get('username'),
+        }
+        for access_key in limited_access_keys_list
+    ]
+
+    markdown_table = tableToMarkdown('PrismaCloud Access Keys', transform_access_keys_list,
+                                     headers=['ID', 'Name', 'Created By', 'Created Timestamp', 'Last Used Time', 'Status',
+                                              'Expires On', 'Role id', 'Role Name', 'Role Type', 'Username'])
+    return CommandResults(
```

Comment:
Please validate that in the next scenario:
1. creating a new access key
2. list all keys

There is no duplication - which means the same access key is appearing twice in the context.